fastlane/Fastfile

```diff
@@ -117,6 +117,32 @@ platform :ios do
     sh("echo GENERATED_BUILD_ID=#{formattedBuildNumber} > #{LOCAL_PATH}/BuildId.xcconfig")
   end
 
+  desc "Fetches the app version number from App Center."
+  lane :appcenter_fetch_version_number do
+    UI.message("\n\n\n=====================================\n Fetch latest version: \n=====================================")
+
+    version = appcenter_fetch_version_number(
+      api_token: ENV["APPCENTER_API_TOKEN"],
+      owner_name: "BraveSoftware",
+      app_name: "brave-ios"
+    )
+
+    UI.message(version.to_s)
+  end
+
+  desc "Fetches the device list from App Center."
+  lane :appcenter_fetch_devices do
+    UI.message("\n\n\n=====================================\n Fetch devices: \n=====================================")
+
+    version = appcenter_fetch_devices(
+      api_token: ENV["APPCENTER_API_TOKEN"],
+      owner_name: "BraveSoftware",
+      app_name: "brave-ios"
+    )
+
+    UI.message(version.to_s)
+  end
```

Comment:
I dont think we need this one, since we aren't doing anything that will require specific devices

Reply:
have added it just for testing and to make sure the API works with the API key